diagnostic.md

```diff
@@ -48,11 +52,15 @@ Record your responses inside the fenced code blocks below each question.
     value `'123'` inside a Route?
 
     ```md
-    <!-- your response here -->
+    params.movie_id ?? not sure
+    so, params.123????
     ```
 
 1.  Inside a template, how do we reference data provided by a Route?
 
     ```md
-    <!-- your response here -->
+    Ember application router maps a URL to a route.
+    An Ember route loads the template and the model.
+    When the model loads, the model hook will return the ember data record,
+    which is then rendered to the template??
```

Comment:
This sounds right, but all we were looking for `model`

We use the model to reference data provided by the route.